dotcom-rendering/scripts/deploy/riff-raff.yaml

```
@@ -3,28 +3,48 @@ regions: [eu-west-1]
 allowedStages:
   - CODE
   - PROD
+templates:
+  cloudformation:
+    type: cloud-formation
+    parameters:
+      amiEncrypted: true
+      amiTags:
+        # Keep the Node version in sync with `.nvmrc`
+        Recipe: dotcom-rendering-ARM-jammy-node-18.17.0
+        AmigoStage: PROD
 deployments:
   frontend-cfn:
-    type: cloud-formation
+    template: cloudformation
     parameters:
       templateStagePaths:
         CODE: DotcomRendering-CODE.template.json
         PROD: DotcomRendering-PROD.template.json
       cloudFormationStackByTags: false
       cloudFormationStackName: rendering
       amiParameter: AMIRendering
-      amiEncrypted: true
-      amiTags:
-        # Keep the Node version in sync with `.nvmrc`
-        Recipe: dotcom-rendering-ARM-jammy-node-18.17.0
-        AmigoStage: PROD
   rendering:
     type: autoscaling
     parameters:
       bucketSsmKey: /account/services/dotcom-artifact.bucket
     dependencies:
       - frontend-static
       - frontend-cfn
+  render-front-cfn:
+    template: cloudformation
+    parameters:
+      templateStagePaths:
+        CODE: DotcomRendering-render-front-CODE.template.json
+        PROD: DotcomRendering-render-front-PROD.template.json
+      cloudFormationStackByTags: false
+      cloudFormationStackName: render-front
+      amiParameter: AMIRenderfront
+  render-front:
+    type: autoscaling
+    parameters:
+      bucketSsmKey: /account/services/dotcom-artifact.bucket
+    dependencies:
+      - frontend-static
+      - frontend-cfn
```

I think we want to depend on the `render-front-cfn` deployment here:
```suggestion
      - render-front-cfn
```